app/controllers/orders/view.js

```diff
@@ -2,21 +2,39 @@ import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
 
 export default Controller.extend({
-  isLoading : false,
-  printThis : service(),
-  actions   : {
+  isLoadingInvoice : false,
+  isLoadingTickets : false,
+  printThis        : service(),
+  actions          : {
     downloadInvoice() {
-      const selector = '.print';
-      const options = {
-        header     : 'Order Invoice',
-        printDelay : 800
-      };
-      this.get('printThis').print(selector, options);
+      this.set('isLoading', true);
+      this.get('loader')
+        .downloadFile(`/orders/invoices/${this.model.order.identifier}`)
+        .then(res => {
+          const anchor = document.createElement('a');
+          anchor.style.display = 'none';
+          anchor.href = URL.createObjectURL(new Blob([res], { type: 'application/pdf' }));
+          anchor.download = 'Invoice.pdf';
```

Comment:
Name it similar to how tickets are named.

Reply:
Done